src/layers/core/point-density-hexagon-layer/point-density-hexagon-layer.js

```diff
@@ -48,13 +53,10 @@ function _needsReProjectPoints(oldProps, props) {
   return oldProps.radius !== props.radius;
 }
 
-function _getCountRange(hexagons) {
-  return [
-    Math.min.apply(null, hexagons.map(bin => bin.points.length)),
-    Math.max.apply(null, hexagons.map(bin => bin.points.length))
-  ];
+function _percentilChanged(oldProps, props) {
```

Comment:
Typo: percentileChanged.

Also why have this outside the layer instead of a method? It deals with layer specific things, like props so looks like it would fit nicely as a method in the layer.